bitnami/postgresql/templates/backup/cronjob.yaml

```diff
@@ -95,6 +95,9 @@ spec:
             {{- if .Values.backup.cronjob.containerSecurityContext.enabled }}
             securityContext: {{- omit .Values.backup.cronjob.containerSecurityContext "enabled" | toYaml | nindent 14 }}
             {{- end }}
+            {{- if .Values.backup.cronjob.resources }}
```

Comment:
Could you add the resourcesPreset section just like in the rest of the objects?

Reply:
Is this the format?
```yaml
          {{- if .Values.readReplicas.resources }}
          resources: {{- toYaml .Values.readReplicas.resources | nindent 12 }}
          {{- else if ne .Values.readReplicas.resourcesPreset "none" }}
          resources: {{- include "common.resources.preset" (dict "type" .Values.readReplicas.resourcesPreset) | nindent 12 }}
          {{- end }}
```